website/src/specs/src/main.js

```diff
@@ -0,0 +1,95 @@
+const changeThemeSelect = document.getElementById('js-changeTheme');
+const customThemeStyleSheet = document.getElementById('js-customTheme');
+const localStorageSelectChoice = localStorage.getItem('selectChoice') || 0;
+const hoverClass = document.querySelectorAll('.js-hoverClass');
+const codeWrapperTabs = document.querySelectorAll('.code-wrapper-tab');
+
+// Store current CSS theme in localStorage
+
+customThemeStyleSheet.setAttribute(
+  'href',
+  changeThemeSelect.options[localStorageSelectChoice].value
+);
+
+if (changeThemeSelect) {
+  changeThemeSelect.value =
+    changeThemeSelect.options[localStorageSelectChoice].value;
+  changeThemeSelect.addEventListener('change', (e) => {
+    customThemeStyleSheet.setAttribute(
+      'href',
+      changeThemeSelect.options[e.target.options.selectedIndex].value
+    );
+    localStorage.setItem('selectChoice', e.target.options.selectedIndex);
+  });
+}
+
+// Hover list of classes to highligh them
+
+function highlightMatches(item) {
+  const queue = [document.getElementById('js-pre')];
+  const word = item.getAttribute('data-value').replace(/\./g, '');
+  let curr;
+
+  while ((curr = queue.pop())) {
+    if (curr.textContent.match(word)) {
+      // eslint-disable-next-line no-loop-func
```

Comment:
Properly fixing this ESLint violation would mean refactoring this code, which isn't worth the effort right now.

Let's decide what to do with all the different sites, then we can clean up this code and test it.